lib_test.py

```diff
@@ -221,16 +236,30 @@ async def test_load_repos_info(mocker):
         project_type=LIBRARY_TYPE,
         web_application_type=None,
         packaging_tool=NPM,
+        versioning_strategy=NPM_VERSION,
+    )
+    expected_lib = RepoInfo(
```

Comment:
Minor nitpick, maybe rename this to `expected_file_library` and also rename `expected_library` to `expected_npm_library`?